dfsclient/src/main/java/org/hpccsystems/dfs/client/RowServiceInputStream.java

```diff
@@ -290,6 +312,24 @@ public void run()
         }
     }
 
+    RestartInformation getRestartInformationForStreamPos(long streamPos)
+    {
+        RestartInformation restartInfo = new RestartInformation();
+
```

Comment:
does this simplify this code a little bit? for (int index = streamPosOfFetches.size() - 1 ; index > -1; index--)